operator/v1/types_network.go

```diff
@@ -750,3 +757,14 @@ const (
 	// IPAMTypeStatic uses static IP
 	IPAMTypeStatic IPAMType = "Static"
 )
+
+type IPSecExternalState string
+
+const (
+	// IPSecExternalStateEmpty is used for defaulting, it is the same as disabled
+	IPSecExternalStateEmpty IPSecExternalState = ""
```

Comment:
also from here

Reply:
I am not sure this can be removed if the enum includes an empty string (I just don't know, I am not sure that giving a name to an empty string is a good idea, but hit is currently a valid value in the enum.